src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +29,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.nextSequenceNumber += 1;
+        System.out.println("Sequence number is " + this.sequenceNumber);
```

Comment:
good that you are using the generated sequence number in this method.
however, you are calling a raw print function here, recollect that there is an object specifically used to do user interaction, try using the sequence number in some class that uses textUi.